index.html

```diff
@@ -159,24 +160,28 @@
 		{
 			name: "AEAD_AES_128_CCM",
 			climit: function(p, l) {
-				return ccm_climit("AEAD_AES_128_CCM", 128, p, l);
+				let l_in_blocks = l / 16;
+				return ccm_climit("AEAD_AES_128_CCM", 128, p, l_in_blocks);
 			},
 			ilimit: function(p, l) {
-				return ccm_ilimit("AEAD_AES_128_CCM", l);
+				let l_in_blocks = l / 16;
+				return ccm_ilimit("AEAD_AES_128_CCM", l_in_blocks);
 			}
 		},
 		{
 			name: "AEAD_AES_128_CCM_8",
 			climit: function(p, l) {
-				return ccm_climit("AEAD_AES_128_CCM_8", 64, p, l);
+				let l_in_blocks = l / 16;
+				return ccm_climit("AEAD_AES_128_CCM_8", 64, p, l_in_blocks);
 			},
 			ilimit: function(p, l, q) {
-				return ccm_ilimit("AEAD_AES_128_CCM_8", l);
+				let l_in_blocks = l / 16;
+				return ccm_ilimit("AEAD_AES_128_CCM_8", l_in_blocks);
 			}
 		},
 	];
 	let sizes = [];
-	for (i = 0; i <= 16; i += 2) {
+	for (i = 4; i <= 16; i += 2) {
```

Comment:
Oh, I thought that 0 might be interesting for cases where you have a tag on every block.  That might happen for something like NTP that has very small messages, but maybe they aren't that small and it's rare for a use case to have a maximum message size that small.

Reply:
The CCM confidentiality bound results in infinity if l = 0 (we divide by l^2). I left it in just for kicks, and added back the [1,2,4,8] values.